### PR TITLE
Update ledger's metadata

### PR DESCRIPTION
**Pull request by CremaFR**
## Related Issue

Soon

### CODE REVIEW

File: src/data/wallets/wallet-data.ts [hunk 1]
```diff
@@ -1153,7 +1153,7 @@ export const walletsData: WalletData[] = [
     firefox: false,
     chromium: false,
     hardware: true,
-    open_source: false,
+    open_source: true,
```
Comment (wackerow) [inline src/data/wallets/wallet-data.ts:1156]: Evidence please.

Comment (CremaFR) [reply src/data/wallets/wallet-data.ts:1156]: Unless this boolean has a different meaning for hardware companies in this list of software wallets, Ledger Live is open source, as can be seen in our GitHub repo: https://github.com/LedgerHQ/ledger-live/. Or even backend should be open sourced? depending on your criteria.

But honestly, I have no interest in debating it. If you want to set it to false, go ahead

File: src/pages/index.tsx [hunk 2]
```diff
@@ -193,7 +192,8 @@ export const getStaticProps = (async ({ locale }) => {
       metricResults,
       rssData: { rssItems, blogLinks },
     },
-    revalidate: REVALIDATE_TIME,
+    // TODO: re-enable revalidation once we have a workaround for failing builds
```
Comment (minimalsm) [inline src/pages/index.tsx:195]: Let's remove the changes unrelated to ledger's metadata from the PR

**Comment by minimalsm** [comment]
Thanks for the PR @CremaFR !

**Comment by minimalsm** [comment]
@all-contributors please add @CremaFR for docs

**Pull request merged**